.changes/config.json

```diff
@@ -4,8 +4,7 @@
     "javascript": {
       "version": true,
       "getPublishedVersion": "npm view ${ pkgFile.pkg.name } version",
-      "publish": "npm publish --access public",
-      "errorOnVersionRange": "4.0.0-alpha.100 - 99.x || ^4.4.0-0 || ^5.0.0-0"
```

Comment:
@cowboyd What was the reason for wanting to prevent those specific version ranges?

Reply:
Not sure... @jbolda ?